backend/corpora/lambdas/api/v1/dataset.py

```diff
@@ -63,17 +63,12 @@ def get_status(dataset_uuid: str, token_info: dict):
     dataset = Dataset.get(db_session, dataset_uuid)
     if not dataset:
         raise ForbiddenHTTPException()
-    collection = Collection.get_collection(
+    authorize_get_collection(
         db_session,
         dataset.collection.id,
-        dataset.collection.visibility,
         owner=owner_or_allowed(token_info),
     )
-    if not collection:
-        raise ForbiddenHTTPException()
-    status = dataset.processing_status.to_dict(remove_none=True)
-    for remove in ["dataset", "created_at", "updated_at"]:
-        status.pop(remove)
+    status = dataset.processing_status.to_dict(remove_none=True, remove_attr=["dataset", "created_at", "updated_at"])
```

Comment:
we are also removing `id` and `dataset_id` in the call in status.py--is it intentional that they are not removed in this similar call?